recipe/meta.yaml

```diff
@@ -23,13 +23,13 @@ requirements:
   build:
     - pkg-config
     - libtool
-    - freetype 2.6.*
+    - freetype 2.7*
```

Comment:
:+1: 

Note that changes like like this one are usually followed by an update in the overall package `pinnings` here:

https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L37

That will trigger automated PRs updating all packages to keep them consistent.